docs/HONEYBADGER_KO.md

```diff
@@ -0,0 +1,193 @@
+# HoneyBadger BFT Protocol
+### Introduction
+기존의 분산 장애 허용 프로토콜(Distributed fault tolerance protocol)은 주로 작은 규모, 단일 관리 도메인(Single administrative domain)을 
+목표로 배포되고 있었습니다.<br>
+그러나 최근에는 새로운 분산 시스템인 *암호화폐(cryptocurrencies)*, *블록체인(blockchains)* 이 등장하면서, 예측이 힘든 큰 규모의 네트워크에 적합하며, 
+악의적인 노드들에 저항성을 갖는 시스템들에 대한 연구들이 진행되었습니다.<br>
+이런 프로토콜은 아래와 같은 점들을 만족해야 합니다.<br>
+* **견고함**<br>
+암호화폐는 성능을 희생해서라도 견고한 보안을 유지하려고 합니다.<br>
+비트코인(Bitcoin)은 분산 시스템의 기준에서 굉장히 나쁜 성능을 갖고 있지만, 기존의 분산 시스템에 비해 훨씬 더 많은 위험에 노출되어 있기 때문에 견고한 시스템인 
+작업 증명(PoW, Proof of Work)을 선택했습니다.<br>
+
+* **지연 대비 처리량 향상**<br>
+기존의 장애 허용 시스템이 성능 최적화에 많은 관심을 보였다면, 블록체인은 원활한 환불 및 롤백 처리를 위해 의도적으로 트랜젝션 처리에 지연을 두기도 합니다.<br>
+
+### HoneyBadgerBFT의 특징
+#### 비동기적인 원자적 브로드캐스트 프로토콜
+대부분의 Byzantine fault tolerant(BFT) 시스템에서는 메세지가 *정해진 시간 안에 도착* 해야 합니다.<br>
```

Comment:
1. Do not enter it and write the sentence in a row.

2. I think it is better to write as `blockchain based cryptocurrency` than just `cryptocurrency`.

Reply:
I can't under stand what you said.